src/pages/BlendPoolPage.tsx

```diff
@@ -130,45 +185,75 @@ export default function BlendPoolPage() {
             <GridExpandingDiv className='w-full min-w-[300px] md:mt-24 md:grid-flow-row-dense'>
-              <PoolInteractionTabs poolData={poolData} />
+              <PoolInteractionTabs
+                poolData={poolData}
+                walletIsConnected={walletIsConnected}
+              />
             </GridExpandingDiv>
           )}
-          <PoolPositionWidget poolData={poolData} />
-          <PoolStatsWidget poolStats={poolStats} />
+          {walletIsConnected && <PoolPositionWidget poolData={poolData} />}
+          <PoolStatsWidget
+            offChainPoolStats={offChainPoolStats}
+            uniswapVolume={uniswapVolume}
+          />
           <PoolPieChartWidget poolData={poolData} />
           <div className='flex flex-col gap-y-6 mt-16'>
-            <WidgetHeading>About Aloe Blend Pool</WidgetHeading>
-            <Text size='M' weight='medium' color={ABOUT_MESSAGE_TEXT_COLOR} className='flex flex-col gap-y-6'>
+            <WidgetHeading>About The Pool</WidgetHeading>
+            <Text
+              size='M'
+              weight='medium'
+              color={ABOUT_MESSAGE_TEXT_COLOR}
+              className='flex flex-col gap-y-6'
+            >
               <p>
-                Placing funds into a Blend Vault will allow Aloe to use Uniswap
-                V3 and yield-earning silos on your behalf.
+                Placing funds into a Blend Pool will allow Aloe smart contracts
+                to use Uniswap V3 and yield-earning silos on your behalf.
               </p>
               <p>
-                When you deposit to the vault, your tokens are pooled together
-                with all other users. Once conditions are right, the vault can
-                be "rebalanced". some tokens get placed in Uniswap, and some in
-                the token's silo. Blend is designed to allocate tokens such that
-                overall value will be split 50/50 between the two tokens, just
-                like Uniswap V2. In Aloe Blend, you earn yield from both Uniswap
-                V3 and the silos, unlike Uniswap V2.
+                When you deposit, your tokens are pooled together with other
+                users'. Once conditions are right, the pool can be rebalanced.
+                During a rebalance, the pool's algorithm redistributes funds
+                between Uniswap and silos to earn the best mix of swap fees and
+                yield. It also tries to keep itself balanced — 50%{' '}
+                {GetTokenData(poolData.token0Address.toLowerCase()).ticker} and
+                50% {GetTokenData(poolData.token1Address.toLowerCase()).ticker},
+                just like Uniswap V2. In the right market conditions, this can
+                massively{' '}
+                <a
+                  href='https://research.paradigm.xyz/uniswaps-alchemy'
+                  target='_blank'
+                  className='underline'
+                  rel='noreferrer noopener'
```

Comment:
Added as having target='_blank' without rel='norefferer noopener' is a security risk.